th2-service/templates/th2-alert-rules.yaml

```diff
@@ -0,0 +1,37 @@
+{{ if .Values.prometheus.operator.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: th2-alert-rules
+  annotations:
+  {{- include "common-annotations" . | nindent 4  }}
+  namespace: {{ .Values.prometheus.operator.serviceMonitor.namespace }}
+  labels:
+    app: {{ template "rabbitmq-ha.name" . }}
+    chart: {{ template "rabbitmq-ha.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+{{- if .Values.rabbitmq.prometheus.operator.serviceMonitor.selector }}
+{{ toYaml .Values.rabbitmq.prometheus.operator.serviceMonitor.selector | indent 4 }}
+{{- end }}
+{{- if .Values.rabbitmq.prometheus.operator.alerts.selector }}
+{{ toYaml .Values.rabbitmq.prometheus.operator.alerts.selector | indent 4 }}
+{{- end }}
```

Comment:
please add appropriate values to prometheus.operator level and remove all relations to rabbitmq chart